### ISSUE TITLE
Cobblestone is invisible on new trade GUI

### ISSUE THREAD

**Issue by Pyromanical** [Wynncraft/Issues#7186]
In the new merchant GUI, the cobblestone in the cobblestone merchant in Olux is invisible.  
![2017-01-16_13 24 23](https://cloud.githubusercontent.com/assets/21978993/21996348/7087d51a-dbef-11e6-96b1-044e64fe6ec1.png)

**Comment by Numbermaniac** [comment]
Is that the WynnPack? Unless there's been a change in the last 12 hours, I don't think the merchant GUI is meant to look like that.

**Comment by Pyromanical** [comment]
Yes, it is the Wynnpack, there was an unannounced update that added the new merchant GUI

**Comment by LordHelmchen98** [comment]
It looks like there is still something. Is it possible to buy the cobblestone?

**Comment by Pyromanical** [comment]
Yup, it works fine, you can buy it, it is just graphically invisible

**Comment by DohdoEXE** [comment]
How did I not find this?

**Comment by DebugMC** [comment]
Another bug: Click the bottom of the gui (bottom row) it has to be under the item you want to buy, the lore will change to the price of the item. (I will provide screenshots later)

**Comment by Moe-Ronickah** [comment]
DebugMC, please make that a separate issue report.
